File: backend/src/main/java/com/example/cinemaweb/dto/SignupRequest.java
package com.example.cinemaweb.dto;

import java.util.Date;

import lombok.Data;

@Data
public class SignupRequest {
    private String fullname;
    private String email;
    private String password;
    private String confirm;
    private Date birthday;
    private String gender;
    private String phoneNumber;
}

File: frontend/src/shared/interfaces/SignupRequest.ts
enum EnumGender {
    male = "Nam",
    female = "Nữ",
    other = "Khác"
}

interface SignupRequest {
    fullname: string;
    email: string;
    password: string;
    comfirm: string;
    birthday: Date;
    gender: EnumGender;
    phoneNumber: string;
}